packages/relay-svm-wallet-adapter/src/adapter.ts

```diff
@@ -77,6 +77,9 @@ export const adaptSolanaWallet = (
         ['Transaction Signature obtained', signature],
         LogLevel.Verbose
       )
+      if (signature && typeof signature === 'string') {
```

Comment:
How would this ever happen? `signAndSendTransaction` should always return a promise that resolves an object with a signature property

Reply:
walletProvider from useAppKitProvider(@reown/appkit/react) returns hash only from walletProvider.signAndSendTransaction function